src/key.rs

```diff
@@ -415,7 +658,7 @@ pub enum Action {
     #[serde(rename = "documents.delete")]
     DocumentsDelete,
     /// Provides access to the [create index](https://docs.meilisearch.com/reference/api/indexes.md#create-an-index) endpoint.
-    #[serde(rename = "indexes.create")]
+    #[serde(rename = "indexes.execute")]
```

Comment:
Is this on the same PR on purpose?

Reply:
No, and it’s wrong I think

Reply:
See this [comment](https://github.com/meilisearch/meilisearch-rust/pull/313#discussion_r953894117)

Reply:
Indeed these are incorrect!